velox/exec/tests/TableEvolutionFuzzer.cpp

```diff
@@ -109,6 +119,18 @@ std::vector<std::vector<RowVectorPtr>> runTaskCursors(
           results.push_back(std::move(result));
         }
         promise.setValue(std::move(results));
+      } catch (VeloxRuntimeError& e) {
+        if (FLAGS_enable_oom_injection &&
+            e.errorCode() == facebook::velox::error_code::kMemCapExceeded &&
```

Comment:
Sometimes a different exception is also allowed as long as there is no crash.  Let's see what we get with this though.